r_code/group_A.R

```diff
@@ -0,0 +1,153 @@
+#############################
+## Load required libraries ##
+#############################
+
+library(DESeq2)
+library(ggplot2)
+library(data.table)
+
+#################################
+## Defining output directories ##
+#################################
+
+## Main output directory:
+OUTPUT_DIR <- file.path("results")
```

Comment:
I agree with Ladislav that it is probably better to name it like nbd24 or something